src/rsz/include/rsz/Resizer.hh

```diff
@@ -234,8 +234,8 @@ class Resizer : public dbStaState
   double wireClkCapacitance(const Corner* corner) const;
   double wireClkHCapacitance(const Corner* corner) const;
   double wireClkVCapacitance(const Corner* corner) const;
-  void estimateParasitics(ParasiticsSrc src);
-  void estimateWireParasitics();
+  void estimateParasitics(ParasiticsSrc src, const char* file_path = "");
+  void estimateWireParasitics(const char* path = "");
```

Comment:
std::string_view is more flexible than char*

Reply:
There is a request to take 
```
 std::map<Corner*, std::ostream*>& spef_files
```
instead of a file_path so that the client can control the output (to a ofstream or a ststream).  The .i could handle setting these up for file based output.

Reply:
Note that  corners could be a subset of all available corners

Reply:
You could keep the write to file version an an overload that uses the other API as its implementation.